### ISSUE TITLE
`qr code` - overlay on scanner

### ISSUE THREAD

**Issue by thedoublejay** [DeFiCh/wallet#558]
#### Why is this needed:
![image](https://user-images.githubusercontent.com/25013382/129283440-4da6fd20-78ff-4223-87a7-98e4e0926825.png)

**Comment by thedoublejay** [comment]
No need for close as it's already available. This ticket is only for overlay

**Comment by ivan-zynesis** [comment]
I find it quite misleading. I mean everyone, every app are doing this, the truth is the scanner work regardless the QR code is centered in the view finder.

**Comment by thedoublejay** [comment]
Yes but it's not a matter of it won't work or not, but a guide especially when the scanner doesn't work.

**Comment by kyleleow** [comment]
@thedoublejay I think the Close button is not available on iOS device
![IMG_7C4B58DD2765-1](https://user-images.githubusercontent.com/40191153/130605234-aac6ce88-33fd-4d4d-a1b1-3f1bf1610c83.jpeg)

**Comment by thedoublejay** [comment]
> @thedoublejay I think the Close button is not available on iOS device
> ![IMG_7C4B58DD2765-1](https://user-images.githubusercontent.com/40191153/130605234-aac6ce88-33fd-4d4d-a1b1-3f1bf1610c83.jpeg)
> 

Sure, the important part is the square overlay
